.gitignore

```diff
@@ -23,6 +23,8 @@
 .vimrc
 aiohttp/_frozenlist.c
 aiohttp/_frozenlist.html
+aiohttp/_helpers.c
+aiohttp/_helpers.html
```

Comment:
@asvetlov it looks like you forgot to update MANIFEST.in.
I wish we could just use a setuptools plugin, which would rely on git to do this automatically.

Reply:
`*.c` files should be included but `*.html` should not.
Not very big problem though: on travis html files are never generated.

Reply:
Fixed by ffa4532e